packit/cli/test.py

```diff
@@ -63,23 +72,32 @@ def test(
         local_project=path_or_url,
     )
 
-    stdout = api.run_local_test(
+    # TODO: Expose options for mock build as CLI flags instead of using hardcoded values.
+    tmt_output = api.run_local_test(
```

Comment:
Same as the other comment in this file. It is a different PR and there is more design talk to be had around it, let's deffer these and discuss them in a different issue/PR.